dist/about.html

```diff
@@ -67,158 +67,179 @@ <h1 class="page-header">About</h1>
 					“Terms of Service; Didn't Read” (short: ToS;DR) is a young project started in June 2012 to help fix the “biggest lie on the web”: almost no one really reads the terms of service we agree to all the time.
 				</p>
 				<p>
-					<strong>Background.</strong> The rough idea behind ToS;DR emerged during the <a href="http://events.ccc.de/camp/2011/wiki/ToS;DR">2011 Chaos Communication Camp</a> near Berlin, with people from <a href="http://unhosted.org">Unhosted</a> a movement to create web apps that give users control over their valuable user data and privacy. In the summer of 2012, <a href="http://hugoroy.eu/index.en.html">Hugo Roy</a> (<a href="http://twitter.com/hugoroyd">@hugoroyd</a>) started the legal analysis and brought the project to life. Since then, more people have joined the team and have contributed through the reviewer community. Ultimately, all the work is transparent and the discussions happen in public. Our work is <a href="thanks.html">funded</a> by non-profits organisations and individual donations and gets released as <a href="http://fsfe.org/about/basics/freesoftware.html">free software</a> and open data.</a>
+					<strong>Background.</strong> The rough idea behind ToS;DR emerged during the <a href="https://events.ccc.de/camp/2011/wiki/ToS;DR" target="_blank" rel="nofollow">2011 Chaos Communication Camp</a> near Berlin, with people from <a href="https://unhosted.org" target="_blank" rel="nofollow">Unhosted</a> a movement to create web apps that give users control over their valuable user data and privacy. In the summer of 2012, <a href="https://hugoroy.eu/index.en.html" target="_blank" rel="nofollow">Hugo Roy</a>
+					(<a href="https://twitter.com/hugoroyd" target="_blank" rel="nofollow"><strong>@hugoroyd</strong></a>) started the legal analysis and brought the project to life. Since then, more people have joined the team and have contributed through the reviewer community. Ultimately, all the work is transparent and the discussions happen in public. Our work is <a href="thanks.html" target="_blank">funded</a> by non-profits organisations and individual donations and gets released as <a href="http://fsfe.org/about/basics/freesoftware.html">free software</a> and open data.</a>
 				</p>
 				<p>
-					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
+					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html" target="_blank">our classification</a>.
```

Comment:
```suggestion
					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
```

There is no need for a target blank on internal sites